web/App.tsx

```diff
@@ -20,8 +20,9 @@ export const GET_USER_DATA = gql`
   }
 `;
 
-const AppBase: React.FC<AppProps> = (props) => {
+const AppBase: React.FC<AppProps> = () => {
   const { data, loading } = useQuery(GET_USER_DATA);
+  console.log(loading, data);
```

Comment:
```suggestion
```

Reply:
we need it just for now so that we don't get linting errors on `data`/`loading` not being used 🙂